src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogSink.cs

```diff
@@ -50,6 +51,27 @@ public DatadogSink(string apiKey, string source, string service, string host, st
             }
         }
 
+        public DatadogSink(string apiKey, string source, string service, string host, string[] tags, DatadogConfiguration config, int queueLimit, int? batchSizeLimit = null, TimeSpan? batchPeriod = null)
+            : base(batchSizeLimit ?? DefaultBatchSizeLimit, batchPeriod ?? DefaultBatchPeriod, queueLimit)
+        {
+            if (config.UseTCP)
```

Comment:
Nice to have: Factorize the common code in both constructors into a single method (From line 57 to line 64)

Reply:
Done